docs/swift/getting-started.md

```diff
@@ -82,31 +82,30 @@ brew install bufbuild/buf/buf
 To configure Buf, scaffold a basic [`buf.yaml`][buf.yaml] by running:
 
 ```bash
-buf mod init
+buf config init
 ```
 
 Next, tell Buf how to generate code by creating a new
 [`buf.gen.yaml`][buf.gen.yaml] file:
 
 ```bash
-touch buf.gen.yaml
+mkdir -p connectrpc/eliza/v1 && touch connectrpc/eliza/v1/eliza.proto
```

Comment:
This looks out of place - to change the `eliza.proto` steps you need to update the section above.